lib/src/layer/shared/layer_projection_simplification/widget.dart

```diff
@@ -0,0 +1,48 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_map/flutter_map.dart';
+import 'package:flutter_map/src/layer/shared/layer_projection_simplification/state.dart';
+
+/// A [StatefulWidget] that includes the properties used by the [State] component
+/// which mixes [ProjectionSimplificationManagement] in
+@immutable
+abstract base class PSSupportedWidget extends StatefulWidget {
```

Comment:
Not a huge acronyms for class names myself and would prefer the long variant because it's not really clear without reading the documentation what this class is for. But you can keep it that way if you prefer this more.